mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/adapters/BankAccountsAdapter.kt

```diff
@@ -0,0 +1,91 @@
+package org.mifos.mobilewallet.mifospay.bank.adapters
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import butterknife.BindView
+import butterknife.ButterKnife
+import org.mifos.mobilewallet.core.domain.model.BankAccountDetails
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.utils.DebugUtil
+import javax.inject.Inject
+
+/**
+ * Created by ankur on 09/July/2018
+ */
+class BankAccountsAdapter @Inject constructor() :
+    RecyclerView.Adapter<BankAccountsAdapter.ViewHolder>() {
+    private var mBankAccountDetailsList: MutableList<BankAccountDetails>?
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val v = LayoutInflater.from(parent.context).inflate(
+            R.layout.item_casual_list,
+            parent, false
+        )
+        return ViewHolder(v)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val bankAccountDetails = mBankAccountDetailsList!![position]
```

Comment:
Using `!!` everywhere defeats the purpose of using Kotlin and taking advantage of its nullability checks. Try using safe calls wherever possible

Reply:
Hi @devansh-299,
I have made necessary changes please review